### PR TITLE
Usage example in README

### PR DESCRIPTION
**Pull request by bunyk**
Added a short example of how to create a microphone stream using this library

### CODE REVIEW

**Comment by MarkKremer** [comment]
Thank you so much for the PR. I'm only seeing this now, whoops.

Have you seen the example in the `examples` directory?

That said, I do think an example in the README is an improvement. The only issue I see is that if the microphone is sent directly to the speaker it can create interference and a loud BEEEEEPPPPP (pun intended) and that's not the best welcome to a repository. That's why the existing example writes it to a file instead. Some solutions could be to write the stream in your README example to a file instead too or put some sort of delay between the microphone input and speaker output.

Are you still active here?

File: README.md [hunk 1]
```diff
@@ -16,6 +16,46 @@ apt-get install portaudio19-dev
...
+	"time"
+
+	"github.com/MarkKremer/microphone"
+	"github.com/faiface/beep"
+	"github.com/faiface/beep/speaker"
+)
+
+func main() {
+	// First, some configuration
+	sampleRate := beep.SampleRate(144100) // Choose sample rate
```
Comment (MarkKremer) [inline README.md:38]:
```suggestion
	sampleRate := beep.SampleRate(44100) // Choose sample rate
```

**Review by MarkKremer — CHANGES_REQUESTED** [review]

**Comment by bunyk** [comment]
Hey, sorry, not so active as I wish I could be. Don't remember already, but probably missed that example folder, or maybe it was not best example for [my use-case](https://github.com/bunyk/fasolasi). Maybe it should be linked or embedded, although it's a bit longer with all that signals and file handling.

If it's not much of an improvement, I'll close this PR, code of the library is clear enough to figure it out anyway.

Also, perhaps you know if there is a way to get microphone input on Mac? This portaudio library has some challenges working with M1 processors.

**Comment by MarkKremer** [comment]
Yeah maybe my example can be made a bit shorter.

I don't know how microphone input works on the Mac so I can't help you with that part. If you figure it out I would like to know so I can point other people that way.

Good luck with your project, it seems cool!

**Pull request closed without merging**